tutorials/i18n/internationalizing_games.rst

```diff
@@ -6,58 +6,57 @@ Internationalizing games
 Introduction
 ------------
 
-Sería excelente que el mundo hablara solo un idioma. Unfortunately for
-us developers, that is not the case. While not generally a big
-requirement when developing indie or niche games, it is also common
-that games going into a more massive market require localization.
-
-Godot offers many tools to make this process more straightforward, so
-this tutorial is more like a collection of tips and tricks.
+*Sería excelente que el mundo hablara solo un idioma.* Unfortunately for
```

Comment:
Would it be possible to change the first sentence into something like this:

---

_Sería excelente que el mundo hablara solo un idioma. (It would be excellent if the world spoke only one language)_

Unfortunately for us developers, that is not the case [...]

---

Because I had to translate that first sentence with a bit of effort to understand what that actually means.